src/__tests__/surveys.test.ts

```diff
@@ -757,33 +779,6 @@ describe('surveys', () => {
         })
     })
 
-    describe('decide response', () => {
```

Comment:
since we're mocking load script we don't need to test whether or not it behaves correctly...